Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -129,19 +153,27 @@ FieldProbe::FieldProbe (std::string rd_name)
             for (int lev = 0; lev < nLevel; ++lev)
             {
                 ofs << m_sep;
-                ofs << "[" << c++ << "]probe_Ex_lev" + std::to_string(lev) + u_map[FieldProbePIdx::Ex];
+                ofs << "[" << c++ << "]part_id" + std::to_string(lev);
+                ofs << m_sep;
+                ofs << "[" << c++ << "]part_Rx_lev" + std::to_string(lev) + " (m) ";
+                ofs << m_sep;
+                ofs << "[" << c++ << "]part_Ry_lev" + std::to_string(lev) + " (m) ";
+                ofs << m_sep;
+                ofs << "[" << c++ << "]part_Rz_lev" + std::to_string(lev) + " (m) ";
```

Comment:
Particles are always on the finest level for WarpX, thus we can skip that for id, x,y,z:
```suggestion
                ofs << "[" << c++ << "]part_id";
                ofs << m_sep;
                ofs << "[" << c++ << "]part_Rx_lev (m) ";
                ofs << m_sep;
                ofs << "[" << c++ << "]part_Ry_lev (m) ";
                ofs << m_sep;
                ofs << "[" << c++ << "]part_Rz_lev (m) ";
```